docs/source/CartesianConfig.rst

```diff
@@ -562,6 +562,11 @@ Formal definition
       <value> in all dicts of the current frame. If a dict lacks <key>,
       it will be created.
 
+   -  A statement of the form <key> ~= <value> sets the value of <key>
+      to <value>, in all dicts of the current frame, but only if <key>
+      does not exist in the dict. If a dict lacks <key>, it will be
+      created.
+
    -  A statement of the form <key> += <value> appends <value> to the
```

Comment:
I think this item is best added after the ``?=`` operator since it relates better to it. The order of the items described here could thus be ``=`` for set, ``+=`` for append, ``<=`` for prepend, then continuing with the conditionals like ``~=`` and ``?=``. In fact, the ``~=`` operator is best placed right after the more regular ``?=`` as it is doing the reverse.

Reply:
Actually, `?=` is not doing the reverse as `~=`, the `?*=` operator group is for regex-match-based assignment, which I'd call it batch assignment for easy understanding, so it is not good to insert the new section into these ones.

The below three places are look to me:
1. after `=` (the current approach)
2. before `?=`
3. at last

Which one do you prefer?

Reply:
Yes, I wrote this suggestion at the time I suggested adding regex to the `~=` operator which no longer makes sense.

In this case it would be best to place it before `?=` (2.) since all first three operators create or modify existing value in some way and are best grouped together as more standard. This could be the fourth operator in this list.

Reply:
Ok, will update accordingly, thanks.